docs/modules/ROOT/pages/tutorial-standalone.adoc

```diff
@@ -0,0 +1,371 @@
+Refer to the https://prometheus.io/docs/alerting/latest/configuration/#receiver[Alertmanager documentation^] for details on configuring this.
+
+.alertmanager.yml
+[source, yaml]
+----
+global: <.>
+
+route: <.>
+  routes:
+    - matchers: <.>
+        - severity =~ (warning|critical) <.>
+      receiver: noop <.>
+      continue: true <.>
+
+receivers:
+  - name: noop <.>
+----
+
+<.> `global` specifies the default settings for all alert receivers.
+<.> `route` is the default route.
+    Alertmanager routes form a tree - alerts enter the tree through this node, and descend through it until they reach a route with `continue: false` set.
+<.> The default route must match every alert, so we define a child route that will only match certain alerts.
+<.> `matchers` specifies which alerts will match this route - in this case, those with a severity of either `warning` or `critical`.
+<.> `receiver` specifies which receiver matching alerts are sent to - in this case we are using `noop`.
+<.> If `continue` is set, the alert will proceed through this route's sibling routes, otherwise it will stop here.
+<.> `receivers` is where we define all receivers. In this case, we only define one, named `noop`, but you will likely define others.
+
+Then, run an Alertmanager container:
+
+[source, console]
+----
+docker run -d \
+  -v $(pwd)/alertmanager/alertmanager.yml:/etc/prometheus/alertmanager/alertmanager.yml \
+  -v /tmp/alertmanager/data:/alertmanager \ # Remove this if you do not want to persist the data.
+  -p 9093:9093 \
+  --network cmos \
+  --name alertmanager \
+  prom/alertmanager
+----
+
+Once it has started you will be able to access its UI on http://localhost:9093.
+It will not do anything yet though, because we need to configure Prometheus to send alerts to it.
+
+Add the following to your `prometheus.yml`:
+
+.prometheus.yml
+[source, yaml]
+----
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+            - alertmanager <.>
+----
+
+<.> This is the name we gave to our Alertmanager container.
+    If they are both on the same Docker network, they can resolve each other by hostname.
+
+Finally, tell Prometheus to reload its config.
+You can do this by sending it a `HUP` signal:
+
+[source, console]
+----
+docker kill -s HUP prometheus
+----
+
+[TIP]
+====
+When previously adding the rules file, it was necessary to stop and restart the Prometheus container.
+However, it is not necessary to restart it only modifying the configuration file.
+This is because the `prometheus.yml` is mounted inside the container, so it will pick up any changes we make to it without needing to restart it.
+We still need to tell it to reload the file, but there's no need for a restart if there are no other changes apart from the configuration file.
+====
+
+== Install Loki
+
+Loki is also configured using a YAML file.
+By default this is located in `/etc/loki/local-config.yaml`, however we will name it `loki.yaml` to avoid confusion.
```

Comment:
`yml` or `yaml`? In the `docker run` we reference it using `yml`

Reply:
Need to standardise on one usage?

Reply:
Possibly